sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```
@@ -874,6 +935,245 @@ public async Task GetBatchSettingWithAllFields()
+            ConfigurationSetting testSetting = CreateSetting("yzabcde", "Contains abc", "yzabcde");
+
+            try
+            {
+                await service.SetAsync(testSetting);
+
+                var selector = new SettingSelector("*abc*");
+
+                ConfigurationSetting[] settings = (await service.GetSettingsAsync(selector, CancellationToken.None).ToEnumerableAsync()).ToArray();
+
+                // There should be at least one key available.
+                Assert.GreaterOrEqual(settings.Length, 1);
+
+                foreach (ConfigurationSetting setting in settings)
+                {
+                    StringAssert.Contains("abc", setting.Key);
+                }
+            }
+            finally
+            {
+                await service.DeleteAsync(testSetting.Key);
+            }
+        }
+
+        [Test]
+        public async Task GetBatchSettingsWithCommaInSelectorKey()
+        {
+            ConfigurationClient service = GetClient();
+
+            ConfigurationSetting abcSetting = new ConfigurationSetting("ab,cd", "comma in key");
+            ConfigurationSetting xyzSetting = new ConfigurationSetting("wx,yz", "comma in key");
+
+            try
+            {
+                await service.SetAsync(abcSetting);
+                await service.SetAsync(xyzSetting);
+
+                var selector = new SettingSelector(key:null);
+                selector.Keys.Add("ab,cd");
+                selector.Keys.Add("wz,yz");
+
+                ConfigurationSetting[] settings = (await service.GetSettingsAsync(selector, CancellationToken.None).ToEnumerableAsync()).ToArray();
+
+                Assert.GreaterOrEqual(settings.Length, 2);
+                Assert.IsTrue(settings.Any(s => s.Key == "ab,cd"));
+                Assert.IsTrue(settings.Any(s => s.Key == "wx,yz"));
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
```

What's the point of catching exception in the test and swallowing its stack trace?

---

Nice catch, thanks!  I had it there for debugging.  Will fix.